src/leap/mail/imap/account.py

```diff
@@ -244,15 +249,15 @@ def select(self, name, readwrite=1):
         """
         name = normalize_mailbox(name)
 
+        if name in self._mailboxes:
+            return defer.succeed(self._mailboxes[name])
```

Comment:
I see where the problem is coming from, but I don't think this is a proper solution.
If several MUAs are manipulating the same mailbox, in the IMAP sense, what one MUA does can interfere with the others. For instance, a select with writeable false, will set the state for another MUA using it.

I'm trying to find a better solution for the global dict of listeners.